lib/views/book.erb

```diff
@@ -0,0 +1,31 @@
+<% if current_user %>
+  
+  Your list of bookings:
+
+  <% @bookings_placed.each do |bookings| %>
+
+    <% if bookings.name == current_user.user_name %>
```

Comment:
This is why I'd use `user` as the relation - much more sensible to relate directly on the model. That way you could do
```erb
<% if booking.user == current_user %>
```